docs/crocks/Identity.md

```diff
@@ -0,0 +1,63 @@
+# Identity
+
+`Identity : a -> a`
+
+It's just a function which returns what you pass into it.
+
+## What a crock!
+
+It converts your `non-crock` into a `crock`
+
+```js
+const frozenPizza = Identity('pizza') //Identity pizza
+const microwave = x => 'microwave ' + x
+const dinner = frozenPizza.map(microwave) //Identity "microwave pizza"
+dinner.value() // "microwave pizza"
+```
+
+| Constructor | Instance |
+|:---|:---|
+| [`of`](#of) | [`ap`](#ap), [`chain`](#chain), [`equals`](#equals), [`map`](#map), [`of`](#of), [`sequence`](#sequence), [`traverse`](#traverse), [`value`](#value) |
+
+## Constructors
+
+### of
+
+```js
+crocks.Identity.of([1,2,10])
+crocks.Identity([1,2,10])
+```
+
```

Comment:
`of` is a very specific function that signals an `Applicative`.
In many cases the normal constructor function is the same thing as `of`.
We still include `of` in those cases so any functions that rely on calling `of` can call it.

So the constructor parameters should be represented by the type siggy in most cases.
For things like `Writer` it takes a `Monoid` and kicks you back a constructor that is fixed to that Monoid like: `Writer (Sum b) a`

Also for Maybe, it also takes an internal UnionType and if it is not the UnionType, it delegates to `Maybe.of` (which is basically `Maybe.Just`)

Good 👁 tho.